misc/bench/src/yz_driver.erl

```diff
@@ -9,6 +9,7 @@
 -include_lib("basho_bench/include/basho_bench.hrl").
 -record(state, {default_field, fruits, pb_conns, index, bucket, iurls, surls}).
 -define(DONT_VERIFY, dont_verify).
+-define(SPACER, 'foo ').
```

Comment:
Total nitpick, but mind switching the prefix on space testing keys to something that makes the purpose clear if you're tracking down a problem later?